README.md

```diff
@@ -1824,4 +1824,134 @@ When you are dealing with exceptions in your code, use `throw new RuntimeExcepti
 
 - It gives the programmer more control over the exception.
 - It gives the programmer the power of logging exceptions.
-- It reduces visibility of the working of your code for potential hackers.
\ No newline at end of file
+- It reduces visibility of the working of your code for potential hackers.
+
+## Unix Directories
+
+For operating systems built on Unix (e.g. Linux and Mac) the highest level of the directory, also called the root folder, is denoted by `/`.
+
+The home folder can be written as `/home/username` (e.g.`/home/jen`) or `~`.
+
+The command `pwd` (print working directory) will print the current directory to the screen.
+
+The command `echo $0` will print the name of the shell you are using.
+
+The command `echo $?` will find the value of the last executed command and return it as a number. If the value exists then a `0` will be returned. If the value does not exist a number other than `0` will be returned (which number depends on the shell you are using).
```

Comment:
I think it's easier (and more accurate? I haven't actually looked this up) to think of `$?` as being the return value of the last program that was executed; and then noting that *all programs return a number* - which is conventionally 0 if everything went well.